Compile Postgres with lz4 and zstd support

vadim2404 opened the pull request:
## Describe your changes
```
CONFIGURE =  'CFLAGS=-O2 -g3' '--enable-debug' '--with-openssl' '--with-uuid=ossp' '--with-icu' '--with-libxml' '--with-libxslt' '--with-lz4' '--with-zstd'
LIBS = -lpgcommon -lpgport -lzstd -llz4 -lxslt -lxml2 -lssl -lcrypto -lz -lreadline -lpthread -lrt -ldl -lm
```
It allows using lz4 as a column compression algorithm 

@knizhnik, I've also added `procps` by default into our containers ;) 

## Issue ticket number and link
https://community.neon.tech/t/request-lz4-zstandard-compression-support/609/2

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics? if so did you add the relevant metrics to the dashboard?
- [x] If this PR requires public announcement, mark it with /release-notes label and add several sentences in this section.

## Checklist before merging

- [ ] Do not forget to reformat commit message to not include the above checklist

hlinnaka reviewed (approved):
> @knizhnik, I've also added procps by default into our containers ;)

+1, but perhaps split that to a separate commit, just for a nicer commit history.

vadim2404 commented:
> > @knizhnik, I've also added procps by default into our containers ;)
> 
> +1, but perhaps split that to a separate commit, just for a nicer commit history.

Split into two
